server/db/migrate/20150216190947_add_rhev_params.rb

```diff
@@ -0,0 +1,16 @@
+class AddRhevParams < ActiveRecord::Migration
+  def change
+    remove_column :fusor_deployments, :rhev_params
+    add_column :fusor_deployments, :rhev_hypervisor_host_id, :integer
```

Comment:
With the current model attributes, is there a particular column or set of columns that the deploy action can use to identify that "this deployment needs to include RHEV"?  For example, will it be sufficient to look for the existence of rhev_engine_host_id or rhev_hypervisor_host_id?

Another option might be to have a deploy_rhev flag, since the user is clicking on the project/product when they create the deployment.